source/PlayerInfoPanel.cpp

```diff
@@ -370,7 +370,7 @@ bool PlayerInfoPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comman
 
 
 
-bool PlayerInfoPanel::Click(int x, int y, int clicks)
+bool PlayerInfoPanel::Click(int /* x */, int /* y */, int clicks)
```

Comment:
Seems like you cherry-picked a commit from `master` ... To resolve conflicts/add commits from the base branch, please use `git merge` with a regular merge commit. I believe you will need to force-push to remove this commit from the PR.

Reply:
Thank you, I removed it. I'll defer updating from master for now.